src/elm/Page/Community.elm

```diff
@@ -611,7 +614,11 @@ viewAction loggedIn metadata maybeDate action =
                         [ if validationType == "CLAIMABLE" then
                             button
                                 [ class ("h-10 uppercase rounded-lg ml-1" ++ claimColors ++ claimSize)
-                                , onClick (OpenClaimConfirmation action.id)
+                                , if isClosed == False then
+                                    onClick (OpenClaimConfirmation action.id)
+
+                                  else
+                                    class ""
```

Comment:
🙃

Reply:
This should be fixed when merging to master. Not sure why it's here, honestly :p